torch/_functorch/aot_autograd.py

```diff
@@ -3416,7 +3429,7 @@ def convert(idx, x):
             # aot_export: ban input metadata mutations for now to keep shared code paths simpler.
             # Keeping .resize_() in the graph will require some work
             # Allowing it but keeping the graph functional will require some calling convention changes.
-            if len([x for x in fw_metadata.input_info if x.mutates_metadata]) != 0:
+            if len([x for x in fw_metadata.requires_grad_info[:fw_metadata.num_mutated_inputs] if x]) != 0:
```

Comment:
hmm, why? (this assert is only meant to be checking for input metadata mutations)

Reply:
I think this was an accident/merge issue